depthai/plugin.cpp

```diff
@@ -0,0 +1,344 @@
+
+#include <cstdio>
+#include <iostream>
+
+#include "opencv2/opencv.hpp"
+
+// Inludes common necessary includes for development using depthai library
+#include "depthai/depthai.hpp"
```

Comment:
These should be searched from the environment I think:
```cpp
#include <opencv2/opencv.hpp>
#include <depthai/depthai.hpp>
```

Reply:
That makes sense I think, since they are installed vs found within the project